### PR TITLE
REFACTOR: Paste data object class

### PR DESCRIPTION
**Pull request by raymonshansen**
Delegate to built-in magic methods where available to simplify code.

### CODE REVIEW

File: pastepwn/core/paste.py [hunk 1]
```diff
@@ -15,27 +15,13 @@ def __init__(self, key, title, user, size, date, expire, syntax, scrape_url, ful
...
+    def set_body(self, body=""):
         self.body = str(body)
 
     def __str__(self):
-        paste_dict = self.to_dict()
-        return str(paste_dict)
+        return str(self.to_dict())
 
-    def __repr__(self):
-        return self.__str__()
```
Comment (d-Rickyy-b) [inline pastepwn/core/paste.py]: Well, should have thought about that one!

File: pastepwn/core/paste.py [hunk 2]
```diff
@@ -15,27 +15,13 @@ def __init__(self, key, title, user, size, date, expire, syntax, scrape_url, ful
         self.full_url = full_url
         self.body = None
 
-    def set_body(self, body):
-        if body is None:
-            body = ""
```
Comment (d-Rickyy-b) [inline pastepwn/core/paste.py]: That's not exactly the same. Could you please change it to:
```python
def set_body(self, body=""):
    self.body = str(body or "")
```

File: pastepwn/core/paste.py [hunk 3]
```diff
@@ -15,27 +15,13 @@ def __init__(self, key, title, user, size, date, expire, syntax, scrape_url, ful
...
-                      'full_url': self.full_url,
-                      'date': self.date,
-                      'key': self.key,
-                      'size': self.size,
-                      'expire': self.expire,
-                      'title': self.title,
-                      'syntax': self.syntax,
-                      'user': self.user,
-                      'body': self.body}
-        return paste_dict
```
Comment (d-Rickyy-b) [inline pastepwn/core/paste.py]: I absolutely did not think about this. Nice one.

File: pastepwn/core/paste.py (hunk 2 above)
Comment (raymonshansen) [reply pastepwn/core/paste.py]: Oh, forgot to account for None and other falsy values

**Pull request merged**